script/tool/test/common/package_command_test.dart

```diff
@@ -763,19 +776,27 @@ packages/b_package/lib/src/foo.dart
       ];
       final RepositoryPackage plugin1 =
           createFakePlugin('plugin1', packagesDir);
-      final RepositoryPackage plugin2 =
-          createFakePlugin('plugin2', packagesDir);
+      createFakePlugin('plugin2', packagesDir);
```

Comment:
Does this do anything?

Reply:
It makes a plugin in the mock in-memory filesystem; it's what makes
```
expect(command.plugins, unorderedEquals(<String>[plugin1.path]));
```

A little later actually useful, because we're asserting that even though there were other packages, they weren't included in the iteration (because the fake diff didn't include any files in them).

The change here is because we are no longer expecting to see `plugin2` in that line, so we don't need a variable containing its path, making it an used variable now (which fails one of our lints).

(Setting up fake plugins/packages before running commands is the common pattern for pretty much all of the tooling tests.)

Reply:
Great, thank you for clarifying